geopandas/tests/test_geom_methods.py

```diff
@@ -870,10 +870,10 @@ def test_rotate(self):
 
         o = Point(0, 0)
         res = self.g4.rotate(angle, origin=o).rotate(-angle, origin=o)
-        assert geom_almost_equals(self.g4, res)
+        assert geom_equals_exact(self.g4, res)
```

Comment:
These are also all missing arguments. I would suggest updating the implementation of `geopandas.tests.util.geom_almost_equals` to use `geom_equals_exact` and then renaming the function to `_geom_almost_equals` to indicate it is private. Then all the usages in `test_geom_methods` can be otherwise unchanged.